index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
+    <div class="template-1">
+
+        <!-- Toolbar Nav -->
+    <nav class="toolbar-container">
+        <!-- Toolbar Name -->
+        <div class="toolbar-name-container">
+            <p class="toolbar-name">Wahab</p>
+        </div>
+        <!-- Toolbar Menu -->
+        <div class="toolbar-menu-container">
+            <img src="assets/Menu.svg" alt="Menu icon">
+        </div>
+    </nav>
```

Comment:
Kindly don't forget to wrap this section in a header tag , this will make your code more organized and easy to read

Reply:
@souad988 corrected it.